.github/workflows/r_package.yml

```diff
@@ -244,7 +245,9 @@ jobs:
       - name: Install packages
         shell: bash
         run: |
-          RDscript${{ matrix.r_customization }} -e "install.packages(c('R6', 'data.table', 'jsonlite', 'knitr', 'markdown', 'Matrix', 'RhpcBLASctl', 'testthat'), repos = 'https://cran.rstudio.com', Ncpus = parallel::detectCores())"
+          R_LIBS="c('R6', 'data.table', 'jsonlite', 'knitr', 'markdown', 'Matrix', 'RhpcBLASctl', 'testthat')"
+          RDscript${{ matrix.r_customization }} \
+            -e "install.packages(${R_LIBS}, repos = 'https://cran.rstudio.com', Ncpus = parallel::detectCores())"
```

Comment:
I'm ok with this for now now. But I think we should eventually just move all these R package installs into a single CI script like `ci/install-r-deps.R`, to reduce duplication. Would you support something like that? I can put up a PR if you agree.

Reply:
Absolutely, it would be great!